Add missing subprogram Send_Message_Comma

tjj2017 opened the pull request:
Note this implementation may not provide the functionality of the
missing code.

Now that this missing subprogram has been added other missing subprograms are causing compile errors.  I will investigate these others.

tjj2017 committed:
Add missing subprogram Send_Message_Comma

Note this implementation may not provide the functionality of the
missing code.

tjj2017 committed:
Created a new gpr file and started creating some shadow files

tjj2017 committed:
Added shadow files and various tweeks to get a successful compilation

tjj2017 committed:
convert gpr and spark index & smf files to unix directory style

martin-cs commented on UKNI/Main.idx:
```diff
@@ -1,59 +1,59 @@
+Calibration_Peak            specification is in Calibration_Peak.ads
+Channel_Types               specification is in Channel_Types.ads
+Count_Range_Check           specification is in Count_Range_Check.ads
+Count_Range_Check           body          is in AVR\Count_Range_Check.adb
```
Should this backslash be a slash?

tjj2017 replied:
Yes I missed a few and they should now be fixed.

martin-cs commented on UKNI/Main.idx:
```diff
@@ -1,59 +1,59 @@
+Region_Of_Interest          specification is in Region_Of_Interest.ads
+Registers                   specification is in registers.ads
+Switch                      specification is in Switch.ads
+Switch                      body          is in AVR\Switch.adb
```
Ditto

tjj2017 replied:
Yes I think I've fixed these too.

martin-cs commented on UKNI/Main.smf:
```diff
@@ -1,55 +1,55 @@
+Toolbox.ads
+Channel_Types.ads
+ADC.ads
+AVR\ADC.adb
```
Another one...

tjj2017 replied:
ditto

martin-cs commented on UKNI/Main.smf:
```diff
@@ -1,55 +1,55 @@
+Toolbox-FWHM.adb
+Calibration.ads
+calibration-gain_adjustment.ads
+AVR\calibration-gain_adjustment.adb
```
Another

tjj2017 replied:
ditto

martin-cs commented on UKNI/Main.smf:
```diff
@@ -1,55 +1,55 @@
+Measurement-Isotopics.adb
+Measurement.adb
+Switch.ads
+AVR\Switch.adb
```
Here.

tjj2017 replied:
ditto

martin-cs commented on UKNI/Main.smf:
```diff
@@ -1,55 +1,55 @@
+Switch.ads
+AVR\Switch.adb
+memory_checks.ads
+AVR\memory_checks.adb
```
Here.

tjj2017 replied:
ditto

martin-cs reviewed (approved):
Looks good to me.  If you could sort the slashes then I think we are good to go.  It might be nice to give / update / amend the documentation on how to build and how to SPARK the code.

tjj2017 committed:
Update the README files

Update README.md to explain the changes for ASVAT.
Update README.txt to decribe how to build and analyse.
Changed some "\" to "/" that I had missed.

tjj2017 committed:
Changed some "\" to "/" that I had missed

tjj2017 commented:
I have now updated the README.md and README.txt files

tjj2017 commented:
I have pushed the changes to my repository.

tjj2017 committed:
Remove references to package System and its children
